File: Templates/express/services/{{cookiecutter.app_folder_name}}/{{cookiecutter.app_name}}.js
{%if cookiecutter.strict == "yes" %}'use strict'{% endif %}
const Express = require('express')
{%- if cookiecutter.body_parser == "yes" %}
const bodyParser = require('body-parser')
{%- endif %}
{%- if cookiecutter.cors == "yes" %}
const cors = require('cors')
{%- endif %}
{%- if cookiecutter.config_file == "yes" %}
const config = require('./config')
{%- endif %}
{%- if cookiecutter.connect_DB == "yes" %}
const db = require('./database/database')
{% endif %}
const app = Express();
{%- if cookiecutter.body_parser  == "yes" %}
app.use(bodyParser.json());
{% endif %}
{%- if cookiecutter.cors == "yes" %}
app.use(cors());
{% endif %}
{%- if cookiecutter.use_controllers == "yes" %}
{%- for i in cookiecutter.controllers_list.list %}
{%- for key, value in i.items() %}
app.use('/{{key}}/', require('./controllers/{{key}}'));
{%- endfor %}
{%- endfor %}
{% endif %}

File: Templates/flask/app_web/{{cookiecutter.app_folder_name}}/{{cookiecutter.app_name}}.py
from flask import Flask{%- if cookiecutter.handle_404 == 'yes' %}, render_template {% endif %}
{%- if cookiecutter.connect_DB == 'yes' %}
from flask_sqlalchemy import SQLAlchemy
from models.models import db
{%- endif %}
{%- if cookiecutter.cors == "yes" %}
from flask_cors import CORS
{%- endif %}
{%- if cookiecutter.use_bp == "yes" %}
{%- for bp in cookiecutter.bp_list.list %}
{%- for key, value in bp.items() %}
from blueprints.{{key}} import {{key}}
{%- endfor %}
{%- endfor %}
{%- endif %}

app = Flask(__name__)
{%- if cookiecutter.cors %}
CORS(app)
{%- endif %}
{% if cookiecutter.config_file == 'yes' %}
# Here goes if theres config file
app.config.from_pyfile('./config.cfg')
{%- endif %}
{% if cookiecutter.connect_DB == 'yes' %}
# Creating an SQLAlchemy instance
db.init_app(app)
{% endif %}
{%- if cookiecutter.use_bp %}
{%- for i in cookiecutter.bp_list.list %}
{%- for key, value in i.items() %}
app.register_blueprint({{key}})
{%- endfor %}
{%- endfor %}
{%- endif %}

#Here goes the endpoinst to be added